coralproject/talk
# [CORL-942] In-Stream Mod rebranding

Notes: 
- More incoming as I'm getting Sam's new buttons in.
- CORL-901 needs to be merged first

## What does this PR do?

Rebrands the in-stream moderation area.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Head to stream as Admin/Mod
- Check moderation caret/dropdown
- New styles!
